### PR TITLE
623 redirect facebook

### LINKED ISSUE #623: [FRONTEND] Make events redirect to facebook
**Is your feature request related to a problem? Please describe.**
While we are still implementing our custom events system, we should make users looking for it go to the current one https://www.facebook.com/UoAsnowsports

Image

Will likely be a note underneath the current WIP

Image

**BEFORE MERGING**

- [ ] Storybooks created where possible
- [ ] Tested with [nock](https://tanstack.com/query/v3/docs/framework/react/guides/testing) if data fetching
- [ ] Tests written for critical interactions
- [ ] PR Reviewed (For non-trivial changes)
- [ ] Changes tested after rebasing on master or merging in master (_hint_: `git fetch origin master:master`, then `git rebase master` or `git merge master`)
- [ ] All required PR checks passing

### PR DESCRIPTION
**Pull request by zlrkw11**
closes #623

### CODE REVIEW

**Comment by zlrkw11** [comment]
image

File: client/src/components/generic/WorkInProgressComponent/WorkInProgressComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import Lock from "@/assets/icons/lock.svg"
+import Link from "next/link"
+const WorkInProgressComponent = () => {
+  return (
+    <div className="flex w-full flex-col items-center justify-center gap-4">
+      <Lock className="fill-dark-blue-100" />
+      <h2 className="italic">Events Page Unavailable</h2>
```
Comment (bcho892) [inline client/src/components/generic/WorkInProgressComponent/WorkInProgressComponent.tsx:7]: Is this still generic if the name is hardcoded?

File: client/src/components/generic/WorkInProgressComponent/WorkInProgressComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      <Link
+        href="https://www.facebook.com/UoAsnowsports"
```
Comment (bcho892) [inline client/src/components/generic/WorkInProgressComponent/WorkInProgressComponent.tsx:9]: What if we wanted to use this on another page?

**Review by bcho892 — CHANGES_REQUESTED** [review]

File: client/src/components/generic/WorkInProgressComponent/WorkInProgressComponent.tsx (hunk 1 above)
Comment (zlrkw11) [reply client/src/components/generic/WorkInProgressComponent/WorkInProgressComponent.tsx:7]: yep, good point. I thought we only need this for events

File: client/src/components/generic/WorkInProgressComponent/WorkInProgressComponent.tsx (hunk 2 above)
Comment (bcho892) [reply client/src/components/generic/WorkInProgressComponent/WorkInProgressComponent.tsx:14]: This should not be part of the generic component

**Review by bcho892 — APPROVED** [review]

**Pull request merged**